Remotion/Web/Core/Res/Themes/NovaViso/HTML/TabbedMenu.css

```diff
@@ -1,319 +1,112 @@
+div.tabbedMainMenu span.tabStripTab:not(.disabled) a:focus,
+div.tabbedMainMenu span.tabStripTabSelected:not(.disabled) a:focus,
+div.tabbedSubMenu span.tabStripTab:not(.disabled) a:hover,
+div.tabbedSubMenu span.tabStripTab:not(.disabled) a:focus,
+div.tabbedSubMenu span.tabStripTabSelected:not(.disabled) a:focus
 {
-  background-color: White;
-  color: Black;
+  height: 100%;
+  width: 100%;
+  padding: 0 var(--remotion-themed-spacing-double);
+  background-color: transparent;
+  color: var(--color-standard-text);
   text-decoration: none;
-}
-
-div.tabbedSubMenu span.tabStripTabSelected a,
-div.tabbedSubMenu span.tabStripTabSelected a:focus
-{
-  border: none;
-  background: none;
-  background-color: White;
-  color: Black;
-}
-
-div.tabbedSubMenu span.tabStripTab a:focus span.anchorBody > img:only-child
-{
   outline: none;
 }
 
-div.tabbedSubMenu span.tabStripTab a:focus span.anchorBody > span
-{
-  text-decoration: none;
-}
-
-div.tabbedSubMenu span.tabStripTabSelected a span.anchorBody > span,
-div.tabbedSubMenu span.tabStripTabSelected a:focus span.anchorBody > span
+div.tabbedMainMenu span.tabStripTab:not(.disabled) a span.anchorBody,
+div.tabbedMainMenu span.tabStripTabSelected:not(.disabled) a span.anchorBody,
+div.tabbedSubMenu span.tabStripTab:not(.disabled) a span.anchorBody,
+div.tabbedSubMenu span.tabStripTabSelected:not(.disabled) a span.anchorBody
 {
-  text-decoration: underline;
+  height: 100%;
+  border-bottom: solid calc(var(--remotion-themed-spacing) * 0.75) transparent;
+  box-sizing: border-box;
+  display: flex;
+  align-items: center;
 }
 
-div.tabbedSubMenu span.tabStripTab.disabled a,
-div.tabbedSubMenu span.tabStripTab.disabled a:hover,
-div.tabbedSubMenu span.tabStripTab.disabled a:active
+div.tabbedMainMenu span.tabStripTab:not(.disabled) a:hover span.anchorBody,
+div.tabbedMainMenu span.tabStripTab:not(.disabled) a:focus span.anchorBody,
+div.tabbedSubMenu span.tabStripTab:not(.disabled) a:hover span.anchorBody,
+div.tabbedSubMenu span.tabStripTab:not(.disabled) a:focus span.anchorBody,
+div.tabbedMainMenu span.tabStripTabSelected:not(.disabled) a span.anchorBody,
+div.tabbedSubMenu span.tabStripTabSelected:not(.disabled) a span.anchorBody
 {
-  color: #AAAAAA;
-  background: none;
-  background-color: White;
-  text-decoration: none;
+  border-bottom-color: var(--color-primary);
 }
 
-div.tabbedSubMenu span.tabStripTab.disabled img
-{
-  opacity: 0.4;
-  filter: progid:DXImageTransform.Microsoft.BasicImage(opacity=0.40),
-          progid:DXImageTransform.Microsoft.BasicImage(grayscale=1);
-}
-
-/* Content area of the tab */
-div.tabbedSubMenu span.anchorBody
-{
-  padding-top: 0;
-  padding-bottom: 0;
-  padding-left: 0.3em;
-  padding-right: 0.3em;
-}
-
-div.tabbedSubMenu div.tabStripTabsPane.empty
-{
-  padding-top: 1.7em;
-}
-
-div.tabbedSubMenu div.clearingPane
+div.tabbedSubMenu span.tabStripTab a:focus span.anchorBody > img:only-child
 {
-  clear: both;
+  outline: none;
 }
```

Comment:
Disabled-style has a bug
![image](https://user-images.githubusercontent.com/6972548/133272362-5db50e06-41c5-4e95-a00e-6ae294a3027b.png)

Reply:
Probably should have been more explict on this, sorry. Please don't make the background light-gray. The background stays the same as the rest of the row, just the font-color gets grayed out. Otherwise, things look strange.

This affects both the main and the sub menu rows and you will need to override the TabStrip style's default for this in TabbedMenu.